MIGRATION.md

```diff
@@ -0,0 +1,114 @@
+# Migrating from `dp-frontend-renderer` to `dp-renderer`
+
```

Comment:
Might be worth adding a small sentence or two summarising what this migration document is, future starters may not have much context of it but be pointed to it.